Description: Do not set install RPATH to /usr/lib by default
             /usr/lib RPATH is disallowed on Debian systems
             and is a bad practice in general
Author: Modestas Vainius <modax@debian.org>
Forwarded: not-needed

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -133,15 +133,6 @@ set(INCLUDE_INSTALL_DIR      "${CMAKE_IN
 set(PKGCONFIG_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig/" CACHE STRING "Base directory for pkgconfig files")
 
 
-# Set up RPATH handling, so the libs are found if they are installed to a non-standard location.
-# By default cmake builds the targets with full RPATH to everything in the build directory,
-# but then removes the RPATH when installing.
-# These two options below make it set the RPATH of the installed targets to all
-# RPATH directories outside the current CMAKE_BINARY_DIR and also the library
-# install directory. Alex
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH  TRUE)
-set(CMAKE_INSTALL_RPATH                ${LIB_DESTINATION} )
-
 if(APPLE)
    set(CMAKE_INSTALL_NAME_DIR ${LIB_DESTINATION})
 endif(APPLE)
